vendor/k8s.io/kubernetes/README.md

```diff
@@ -0,0 +1,70 @@
+# Kubernetes
```

Comment:
I typically combine the godep save and this commit into a single commit to avoid the 100kloc deletion and re-addition